utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -11,7 +11,8 @@ private const val CUSTOM_IMG_FILE_PATH_ATTRIBUTE = "filepath-with-value"
 private const val REPLACE_IMG_FILE_PATH_ATTRIBUTE = "src"
 
 /** Html Parser for android TextView to parse Html tag. */
-class HtmlParser(private val context: Context, private val entityType: String, private val entityId: String) {
+class HtmlParser(private val context: Context, private val entityType: String, private val entityId: String,
```

Comment:
Optional Nit: This can be made more readable by bringing each object to next line.

Reply:
This is also part of https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break (breaking at a higher syntactic level).

Here, you broke the line mid-way through the argument list, but the higher syntax is the beginning of the argument list. Thus, we should break at the ``(`` first, and then if all arguments don't fit then we should break along every argument at the ``,``.

Reply:
Fixed it.